source/Dgame/Graphic/Color.d

```diff
@@ -29,6 +29,462 @@ import derelict.sdl2.types;
+    static immutable T Cyan4                = T(  0, 139, 139);
+    static immutable T Darkblue             = T(  0,   0, 139);
+    static immutable T Darkcyan             = T(  0, 139, 139);
+    static immutable T Darkgoldenrod        = T(184, 134,  11);
+    static immutable T Darkgoldenrod1       = T(255, 185,  15);
+    static immutable T Darkgoldenrod2       = T(238, 173,  14);
+    static immutable T Darkgoldenrod3       = T(205, 149,  12);
+    static immutable T Darkgoldenrod4       = T(139, 101,   8);
+    static immutable T Darkgray             = T(169, 169, 169);
+    static immutable T Darkgreen            = T(  0, 100,   0);
+    static immutable T Darkgrey             = T(169, 169, 169);
+    static immutable T Darkkhaki            = T(189, 183, 107);
+    static immutable T Darkmagenta          = T(139,   0, 139);
+    static immutable T Darkolivegreen       = T( 85, 107,  47);
+    static immutable T Darkolivegreen1      = T(202, 255, 112);
+    static immutable T Darkolivegreen2      = T(188, 238, 104);
+    static immutable T Darkolivegreen3      = T(162, 205,  90);
+    static immutable T Darkolivegreen4      = T(110, 139,  61);
+    static immutable T Darkorange           = T(255, 140,   0);
+    static immutable T Darkorange1          = T(255, 127,   0);
+    static immutable T Darkorange2          = T(238, 118,   0);
+    static immutable T Darkorange3          = T(205, 102,   0);
+    static immutable T Darkorange4          = T(139,  69,   0);
+    static immutable T Darkorchid           = T(153,  50, 204);
+    static immutable T Darkorchid1          = T(191,  62, 255);
+    static immutable T Darkorchid2          = T(178,  58, 238);
+    static immutable T Darkorchid3          = T(154,  50, 205);
+    static immutable T Darkorchid4          = T(104,  34, 139);
+    static immutable T Darkred              = T(139,   0,   0);
+    static immutable T Darksalmon           = T(233, 150, 122);
+    static immutable T Darkseagreen         = T(143, 188, 143);
+    static immutable T Darkseagreen1        = T(193, 255, 193);
+    static immutable T Darkseagreen2        = T(180, 238, 180);
+    static immutable T Darkseagreen3        = T(155, 205, 155);
+    static immutable T Darkseagreen4        = T(105, 139, 105);
+    static immutable T Darkslateblue        = T( 72,  61, 139);
+    static immutable T Darkslategray        = T( 47,  79,  79);
+    static immutable T Darkslategray1       = T(151, 255, 255);
+    static immutable T Darkslategray2       = T(141, 238, 238);
+    static immutable T Darkslategray3       = T(121, 205, 205);
+    static immutable T Darkslategray4       = T( 82, 139, 139);
+    static immutable T Darkslategrey        = T( 47,  79,  79);
+    static immutable T Darkturquoise        = T(  0, 206, 209);
+    static immutable T Darkviolet           = T(148,   0, 211);
+    static immutable T Deeppink             = T(255,  20, 147);
+    static immutable T Deeppink1            = T(255,  20, 147);
```

Comment:
What is the point of a color which has the suffix *1* but the same value? Also, a numeric suffix looks like bad design if you ask me.

Reply:
I forgot pygame has duplicates. I just wrote a script to pull them out. Since they are normal web colors. I just got use to numbers because pygame has them built in. I will remove them.